src/test/features/precendence.tact

```diff
@@ -0,0 +1,23 @@
+import "@stdlib/deploy";
+
+contract PrecendenceTester with Deployable {
+    init() {
+
+    }
+
+    get fun test1(): Int {
+        return 5 & 6 | 1 << 5 + 11 * 3 % 12 >> 11;
```

Comment:
we should also test a different order for `&` and `|`, because if we parse it from left to right it's expected we get `(5 & 6) | 1`, let's have also something like `x | y & z`